### ISSUE TITLE
Evaluation and nested entities

### ISSUE THREAD

**Issue by leodotnet** [tticoin/LSTM-ER#11]
Dear Makoto,

I have 2 questions.

1. After I run the test, ann prediction files will be created. However, I cannot find any evaluation script to generate the evaluation result. Do you have it?

2. I notice you mentioned you removed nested entities and relations. Do you mean you discard all the overlapping entities (head span) and relations which contain overlapping entities after reading data? 

Thanks a lot.

Best Regards,
Li Hao

**Comment by matteocannaviccio** [comment]
I agree with Li Hao, scripts to generate results from ***.pred.ann** files would be useful. 

Also, in the paper you claim to follow the settings by (Li and Ji, 2014) for the experiment on ACE05. Does it means that you eliminate "implicit" relations from the system output (i.e. the ones that are implicitly correct via coreference links)?

Thanks for your answer.

Best,
Matteo Cannaviccio
